src/components/action-bar/examples/action-bar-floating-with-expand-button.tsx

```diff
@@ -0,0 +1,65 @@
+import { Component, h, State } from '@stencil/core';
+import { ActionBarItem, ListSeparator } from '@limetech/lime-elements';
+
+/**
+ * Floating action bar with expand button
+ *
+ * Some designs may require a floating action bar with an expand button.
+ * It can be useful if action bar is covering important content.
+ * To make the action bar expandable, set the `isExpandable` prop to `true`.
+ *
+ */
+@Component({
+    tag: 'limel-example-action-bar-floating-expand',
+    shadow: true,
+    styleUrl: 'action-bar-floating.scss',
+})
+export class ActionBarFloatingExpandExample {
+    @State()
+    private actionBarItems: Array<ActionBarItem | ListSeparator> = [
```

Comment:
move above the `@Component({`, this can't be a state since there is nothing to update. I see that we have mentioned this in many components and better to create a new file called `action-bar-items` and reuse it. Of course this can be a refactor work. What do you think? 

```suggestion
    const actionBarItems: Array<ActionBarItem | ListSeparator> = [
```